simulator.js

```diff
@@ -13,11 +13,16 @@
 
 'use strict';
 
+global.Config = require('./config/config.js');
+global.Tools = require('./tools.js').includeMods();
+global.toId = Tools.getId;
```

Comment:
I'm not sure we want to override the globals here. We should probably only do this after detecting a child process.